tutorials/time-series-visualization-with-altair/tutorial.ipynb

```diff
@@ -0,0 +1,2133 @@
+{
```

Comment:
<p>Suggested change:</p><p><strong>Earth Engine</strong> is used to access, clean, and reduce large amounts of spatiotemporal data, <strong>pandas</strong> is used to analyze and organize the results, and</p><p> </p><p>Suggested change:</p><p><strong>Note:</strong> This notebook demonstrates an analysis template and interactive workflow that is appropriate for a certain size of dataset, but there are limitations to interactive computation time and server-to-client data transfer size imposed by Colab and Earth Engine.  To analyze even larger datasets, you may need to modify the workflow to <a href="https://developers.google.com/earth-engine/exporting#exporting-tables-and-vector-data" target="_blank">export</a> <code>featureCollection</code> results from Earth Engine as static assets and then use the static assets to perform the subsequent steps involving Earth Engine table formatting, conversion to pandas DataFrame, and charting with Altair.</p><p> </p><p>It would be better if the export link pointed to a Python export example:</p><p><a href="https://developers.google.com/earth-engine/python_install#exporting-data" target="_blank">https://developers.google.com/earth-engine/python_install#exporting-data</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done.</p><p> </p><p>Done.</p>